### PR TITLE
Add tests for jenkins_worker role.

### PR DESCRIPTION
**Pull request by benpatterson**
@jzoldak @clytwynec

### CODE REVIEW

File: playbooks/roles/jenkins_worker/tasks/test_jenkins_worker.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
+---
+# Tests for this role
+
+
+# Set up #
+- name: Create test file
```
Comment (jzoldak) [inline playbooks/roles/jenkins_worker/tasks/test_jenkins_worker.yml]: Why do you need to create a test file? Explain in comments.

**Comment by benpatterson** [comment]
@jzoldak I added a bit more description, as well as tests for the github python scripts.

File: playbooks/roles/jenkins_worker/tasks/test.yml [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+  register: testfile
+
+
+# Tests #
+- name: Verify java cmd is using v 1.7
+  shell: java -version
+  register: java_version
+- assert:
+    that:
+      - "'1.7.0' in java_version.stderr"
```
Comment (clytwynec) [inline playbooks/roles/jenkins_worker/tasks/test.yml:22]: This goes to stderr not stdout?

Comment (benpatterson) [inline playbooks/roles/jenkins_worker/tasks/test.yml:22]: Believe it or not, yes. The java -version command exits with 0, but this data is fed through stderr.

**Comment by jzoldak** [comment]
:thumbsup:

**Comment by clytwynec** [comment]
:+1:

**Comment by benpatterson** [comment]
@jzoldak @clytwynec Just a heads up I've removed the rbenv test since it'll fail. I added a ticket to fix the underlying issue.

But I'll merge this without that test today.

**Pull request merged**